tools/FhirAnalyzer/LogInterpolation.cs

```diff
@@ -0,0 +1,111 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace FhirAnalyzer
```

Comment:
Did you consider adding this to the shared components repo? Seems like this rule could be useful to dicom as well

Reply:
I did, and I can't figure out why it's not working, despite me doing same thing.
I agree what it's would be nice to share, but it's a bit of work to do.

I would like to commit this one in, and have issue to move it to shared component repo, than this PR just hanging here.